homework2.py

```diff
@@ -0,0 +1,428 @@
+# exercise 5
+'''
+1. write a Python program to find those numbers
+which are divisible by 7 and multiple of 5, between 1500 and 2700 (both included).
+'''
+
+# code up your solution here
+for i in range(1500,2700):
+	if i%7==0 and i%5==0:
+		print(i)
+
+'''
+2. Write a Python program to count the number of even and odd numbers from a series of numbers.
+'''
+num_list = [1,2,3,4,5,6,3,4,1,12,4,6,745,34352,123,6,5,3,3,432,3.1415,5.23]
+odd = []
+even = []
+other = []
+
+for i in num_list:
+    if i%2==1:
+        odd.append(i)
+    elif i%2==0:
+        even.append(i)
+    else:
+        other.append(i)
+
+num_list2 = {'odd':odd, 'even':even, 'other':other}
+print(num_list2)
+
+odd1=0
+even1=0
+
+for i in num_list:
+    if i%2==1:
+        odd1+=1
+    elif i%2==0:
+        even1+=1
+
+print(odd1)
+print(even1)
+
+'''how to use comprehensions to do this'''
+
+# example: numbers = (1, 2, 3, 4, 5, 6, 7, 8, 9)
+# output:
+# >>> Number of even numbers : 4
+# >>> Number of odd numbers : 5
+
+# code up your solution here
+
+
+
+'''
+3. Write a Python program which iterates the integers from 0 to 50.
+-For multiples of three print "Fizz" instead of the number
+-For the multiples of five print "Buzz".
+-For numbers which are multiples of both three and five print "FizzBuzz".
+'''
+
+for i in range(0,51):
+    if i%5==0 and i%3==0 and i!=0:
```

Comment:
the original expected output actually have `FizzBuzz` for 0 so no need for `i!=0`